projects/storefrontlib/cms-components/product/current-product.service.ts

```diff
@@ -55,4 +55,13 @@ export class CurrentProductService {
       .getRouterState()
       .pipe(map((state) => state.state.params['productCode']));
   }
+  getRealTimeStock(
+    productCode: string,
+    unit: string
+  ): Observable<{ quantity: string; availability: string }> {
+    return this.productService.getRealTimeStockDatafromService(
```

Comment:
MAJOR: can we change the method name to 'getRealTimeStock' . Everywhere lets name it as 'getRealTimeStock'  and in adapter class lets make it 'loadRealTimeStock'

Reply:
Done